src/nnet3/nnet-component-itf.cc

```diff
@@ -323,10 +323,8 @@ std::string NonlinearComponent::Info() const {
   if (InputDim() == OutputDim()) {
     stream << Type() << ", dim=" << InputDim();
   } else {
-    // Note: this is a very special case tailored for class NormalizeComponent.
     stream << Type() << ", input-dim=" << InputDim()
-           << ", output-dim=" << OutputDim()
-           << ", add-log-stddev=true";
+           << ", output-dim=" << OutputDim();
```

Comment:
this change is fine, but please add a version of this function in class RenormalizeComponent, that prints the info and prints add-log-stddev if true.